lib/imapSimple.js

```diff
@@ -384,6 +384,38 @@ ImapSimple.prototype.delFlags = function (uid, flags, callback) {
     });
 };
 
+/**
+ * Deletes the specified message(s).
+ *
+ * @param {string|Array} uid The messages uid
```

Comment:
Try this for description: "The uid or array of uids indicating the messages to be deleted"